### DISCUSSION TITLE
Best practice for EJBCA-CE 8.0 docker migration from Ubuntu 22.04.2 LTS

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#532] [General]
Hi All,

Can anyone guide what shall be the best practice to migrate ejbca-ce 8.0 with docker installation from Ubuntu 22.04.2 LTS to higher stable version. Secondly can we move the same to Red Hat 8 enterprise version also.

I have front end on docker while DB on separate machine with mysql 8.0. If there are any stepwise guide available, then I shall be very grateful. 

Highly appreciate if the community can suggest the best option for me keeping the security and performance in mind and also that my database doesn't get disturb since I have root CAs, intermediates CAs and many entities issued already.

**Comment by merryo11** [comment]
any update from the community

**Comment by svenska-primekey** [comment]
Install docker on the new host, and deploy the EJBCA container pointing to the same DB. With the DB on a separate host there is nothing to do there. You can run the old instance and the new instance at the same time with the DB external. Containers make life easy.
